components/admin_console/plugin_management/plugin_management.tsx

```diff
@@ -722,7 +777,9 @@ export default class PluginManagement extends AdminSettings {
         );
     }
 
-    renderRemovePluginModal = ({show, onConfirm, onCancel}) => {
+    renderRemovePluginModal = (
+        {show, onConfirm, onCancel}:
```

Comment:
How come we need both these lines? The one below should take care of the typing.